### ISSUE TITLE
docs: broken link "partials/circular" -> Make it easier to contribute to the docs

### ISSUE THREAD

**Issue by leonardpauli** [amark/gun#710]
see https://gun.eco/docs/API#allowed-types

the links in the first li are broken:

```
<li><code>objects</code>:
<a href="Partials-and-Circular-References-(v0.2.x">partial</a>),
<a href="Partials-and-Circular-References-(v0.2.x">circular</a>#circular-references), and nested</li>
```

Searched a bit but ~couldn't find the page~ ... in wiki: https://github.com/amark/gun/wiki/Partials-and-Circular-References-(v0.2.x)

---

Also, how to edit..? Found https://gun.eco/docs/Contributing#contribute-to-the-documentation but can't find the proclaimed "Edit this page". Also... ohh https://github.com/gundb/gun-site/blob/master/docs/navigation.json there we go, docs has its own repo after all!

**TL;DR: After finding something to fix in the docs, it's a bit "hard" to actually do it**

**Comment by amark** [comment]
@leonardpauli SO DELIGHTFUL TO HEAR FROM YOU AGAIN!!! :D :D :D :) :) :) :)

How have you been? What have you been up to?

Apologies for the delay, I was gone on a vacation when you opened this.

You are correct that adding `/wiki` to the GitHub URL pulls up the wiki that powers the docs. The edit button for content is in top right of the wiki page (or you can just add _edit like https://github.com/amark/gun/wiki/Partials-and-Circular-References-(v0.2.x)/_edit )

The navigation is stuck in a real git, not a wiki. Which makes it harder - you have to fork the repo and make a pull request :( but at least it'll then treat you as an official collaborator on the docs site repo :) !!!

(I think github should show you a little pencil icon in the json file on the repo which if you click will automatically fork in the web UI and let you edit... not sure though, worse case is real git command line :P lol)

**Comment by amark** [comment]
Ok, I fixed the broken link, now https://gun.eco/docs/Partials-and-Circular-References (no parenthesis which tripped up and broke the parser). Thanks!

To anybody seeing this, please make use of the GitHub wiki to _edit pages and PR to navigation.json ! Mucho appreciated.
